### ISSUE TITLE
1.4.11 Understanding: a checkbox focus example image does not match its accompanying text

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag#2190]
### Currently

Understanding 1.4.11 provides an example of "Checkbox - Subtle focus style - fail". The image in this table row shows a focus indicator colored #979797 with a 2.9:1 contrast with its adjacent white background (checkbox-example5.png). However, the accompanying text says "In this case, the gray (#AAA) indicator has an insufficient ratio of 2.3:1 with the white (#FFF) adjacent background."

### Proposed

A 2.3:1 ratio makes a better illustration than a 2.9:1 ratio, so keep the text and change the image.

**Comment by alastc** [comment]
Thanks @mitchellevan, that's odd, perhaps the colour got converted when I saved it. I'll have a look.
